src/applications/vaos/services/mocks/index.js

```diff
@@ -508,7 +508,7 @@ const responses = {
     });
   },
   'GET /vaos/v2/epsApi/referralDetails/:referralId': (req, res) => {
-    const referrals = referralUtils.createReferrals(3, '2024-12-02');
+    const referrals = referralUtils.createReferrals(3, '2024-12-02', 1);
```

Comment:
not part of your changes, but we should probably not hard code this date

Reply:
Agree